components/ui/AppHelpfulResourcesSection.vue

```diff
@@ -1,10 +1,11 @@
 <template>
   <section class="app-helpful-resources-section">
     <h2 class="copy__title" v-text="title" />
-    <div class="app-helpful-resources-section__resources">
+    <div class="bx--row">
       <AppDescriptionCard
         v-for="resource in resources"
         :key="resource.title"
+        class="bx--col-lg-4 bx--col-md-4 app-helpful-resources-section__card"
```

Comment:
As discussed offline, since `bx--col-padding` doesn't appear to do exactly the same (it add a little more space to the edges of the elements as well), we decided to leave the implementation as is and address it in the future with a more general refactor of recurrent classes (#1677).